### PR TITLE
Cut 0.78.1

### PR DESCRIPTION
**Pull request by simonpasquier**
## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [Prometheus-Operator testing guidelines](../TESTING.md) for recommendations about automated tests.

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by nicolastakashi — APPROVED** [review]

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -25,6 +25,11 @@ jobs:
         uses: actions/checkout@v4
       - name: Import environment variables from file
         run: cat ".github/env" >> $GITHUB_ENV
+      - name: Reclaim disk space
```
Comment (nicolastakashi) [inline .github/workflows/publish.yaml:28]: clever hehe

Comment (simonpasquier) [reply .github/workflows/publish.yaml:28]: I would prefer if the GH action ubuntu image was smaller :-/

Comment (nicolastakashi) [reply .github/workflows/publish.yaml:28]: Can't we use maybe a custom image?

Comment (simonpasquier) [reply .github/workflows/publish.yaml:28]: Apparently not and the disk space seems to be a recurrent issue with GitHub runners.

**Pull request merged**